### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by rodrigopenayo98**
## Project Description for Pull Request:

In this project, I've applied advanced SQL concepts to manage a vet clinic's database. I used transactions to update and delete records while employing aggregate functions and GROUP BY clauses to answer analytical questions.

## Project Highlights:

- Added a new `species` column to the `animals` table.
- Inserted various animal records with detailed information.
- Utilized transactions to update and manage data consistently.
- Answered analytical queries using aggregate functions and GROUP BY.
- Captured key screenshots to showcase query results and transaction outcomes.

## Screenshots:
Correction 1
Correction 2
Correction 3
Correction 4

Screenshot 1
Screenshot 2
Screenshot 3
Screenshot 4
Screenshot 5
Screenshot 6
Screenshot 7
Screenshot 8
Screenshot 9
Screenshot 10
Screenshot 11
Screenshot 12
Screenshot 13
Screenshot 14
Screenshot 15
Screenshot 16
Screenshot 17
Screenshot 18

### CODE REVIEW

File: queries.sql
Comment (joskalenda) [inline queries.sql:1]: - [x] You are missing the very first point of the queries, you are asked to do this as stated in the image below..., please make sure you have insert this and resubmit your work

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @rodrigopenayo98 , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ].. ✅ Great readme file
- [ ].. ✅ Clean code

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql
Comment (rodrigopenayo98) [reply queries.sql:1]: @joskalenda  thank you for the correction! I already added the query that I was missing!

**Review by mwanawabangona — APPROVED** [review]
Hi @rodrigopenayo98 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**